common.go

```diff
@@ -16,33 +16,58 @@ import (
 // Logger used to log any output
 type Logger = log.Logger
 
+const (
+	MYSQL_DRIVER    = "mysql"
+	SQLITE_DRIVER   = "sqlite3"
+	POSTGRES_DRIVER = "postgres"
```

Comment:
Let's rename the constants to:
- DriverMySQL
- DriverSQLite
- DriverPostgres

Reply:
Also if you expose them as public, please add a go comments.

Reply:
Updated.